lib/pluginmanager/update.rb

```diff
@@ -6,6 +6,7 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   REJECTED_OPTIONS = [:path, :git, :github]
 
   parameter "[PLUGIN] ...", "Plugin name(s) to upgrade to latest version", :attribute_name => :plugins_arg
+  option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
```

Comment:
No, this option is to skip a validation process done later in the code, you can see it https://github.com/elastic/logstash/blob/master/lib/pluginmanager/update.rb#L69 where there is a warning when updating between major versions. 

The code used there ```rubygems/gems``` only works for rubygems, not for other gem repositories.